pkg/descheduler/controllers/migration/evict.go

```diff
@@ -44,6 +44,12 @@ func (r *Reconciler) Evict(ctx context.Context, pod *corev1.Pod, evictOptions fr
 		return false
 	}
 
+	if sev1alpha1.PodMigrationJobMode(r.args.DefaultJobMode) == sev1alpha1.PodMigrationJobModeReservationFirst &&
+		pod.Spec.SchedulerName != "koord-scheduler" {
```

Comment:
We should support configuring different scheduler names. The scheduler name corresponding to the koord-scheduler component is not necessarily koord-scheduler

Reply:
done